internal/proto/readers/v1/readers.proto

```
@@ -0,0 +1,62 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package readers.v1;
```

Buf (1.47.2)

Files with package "readers.v1" must be within a directory "readers/v1" relative to root but were in directory "internal/proto/readers/v1".

(PACKAGE_DIRECTORY_MATCH)